tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: SAR basics
+description: Introduction to SAR basics using Sentinel-1 in GEE.
+author: glemoine62
+tags: SAR, Sentinel-1, Copernicus, backscattering coefficients, GRD, polarization
+date_published: 2020-08-20
+---
+<!--
+Copyright 2019 The Google Earth Engine Community Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+This tutorial introduces the basics for Sentinel-1 use in GEE. It will illustrate basic SAR terminology and demonstrate data selection and visualization.
```

Comment:
Maybe link to the S1 page in the catalog for context?

Reply:
link added